### DISCUSSION TITLE
Default Value

### DISCUSSION THREAD

**Discussion by jacksonsophat** [pocketbase/pocketbase#2208] [Q&A]
Is it possible to set a default value if not provide. I hope to set an avatar image to a default image if the user is not provided.

**Comment by ganigeorgiev** [comment]
For default avatar you can show a placeholder image if the avatar field is empty.

If you want to upload a default image, you can upload it client-side with the record submission OR if you want to do it server-side, you'll have to [use PocketBase as framework](https://pocketbase.io/docs/use-as-framework/) and attach to a [before* model or record hook](https://pocketbase.io/docs/event-hooks/).
